### PR TITLE
Using base64 to encrypt the data to base64 while installing the helm chart

### PR DESCRIPTION
**Pull request by Anurag-30**
#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://developercertificate.org) signed
- [x] Chart Version bumped (if the pr is an update to an existing chart)
- [x] Variables are documented in the README.md
- [ ] Title of the PR starts with chart name (e.g. `[fluentd-elasticsearch]`)

### CODE REVIEW

**Comment by monotek** [comment]
Please add the changes requested in: https://github.com/kiwigrid/helm-charts/pull/401

By the way...
You can just change the pr by pushing into the branch again. 
No need to open a new pr :)

**Review by monotek — APPROVED** [review]

**Review by monotek — CHANGES_REQUESTED** [review]
Please fix yaml lint: https://github.com/kiwigrid/helm-charts/pull/402/checks?check_run_id=1153090800

and sign dco: https://github.com/kiwigrid/helm-charts/pull/402/checks?check_run_id=1153089862

**Comment by Anurag-30** [comment]
> Please fix yaml lint: https://github.com/kiwigrid/helm-charts/pull/402/checks?check_run_id=1153090800
> 
> and sign dco: https://github.com/kiwigrid/helm-charts/pull/402/checks?check_run_id=1153089862

can you help me fixing this. I ran helm lint in local and it's working fine. The log said remove trailing spaces I have done that yet it exits with an error

**Comment by monotek** [comment]
fixed the values by adding quotes but you have to fix the dco: https://github.com/kiwigrid/helm-charts/pull/402/checks?check_run_id=1162981244

**Comment by monotek** [comment]
@Anurag-30 
Could you please follow the instructions of:  
https://github.com/kiwigrid/helm-charts/pull/402/checks?check_run_id=1162981244

**Comment by Anurag-30** [comment]
> https://github.com/kiwigrid/helm-charts/pull/402/checks?check_run_id=1162981244

I have followed the instructions and pushed it

**Comment by monotek** [comment]
What happenend here? There are changes for 3 different charts now?

**Review by monotek — CHANGES_REQUESTED** [review]
Please cleanup the changes for other charts.

File: charts/spring-cloud-config-server/values.yaml [hunk 1]
```diff
@@ -58,5 +58,5 @@ config:
   gitSearchpath: "{application}"
 
 secrets:
-  gitUsername:
-  gitPassword:
+  gitUsername:
```
Comment (monotek) [inline charts/spring-cloud-config-server/values.yaml:61]: you've deleted the quotes i've added to fix the linter...

**Review by monotek — CHANGES_REQUESTED** [review]

File: charts/spring-cloud-config-server/values.yaml (hunk 1 above)
Comment (Anurag-30) [reply charts/spring-cloud-config-server/values.yaml:61]: Sorry @monotek my bad

**Review by monotek — CHANGES_REQUESTED** [review]
I've added the secret part in https://github.com/kiwigrid/helm-charts/pull/410.
To merge the readme change please rebase and sign your commits.

**Pull request closed without merging**